.jenkins.yml

```diff
@@ -34,8 +34,9 @@ target:
  - cocoapods-osx
  - cocoapods-ios
  - cocoapods-ios-dynamic
- - cocoapods-ios-subdependency
  - cocoapods-watchos
+ - cocoapods-tvos
```

Comment:
Would be nice to distinguish which targets are building the framework and which are installation test